### ISSUE TITLE
Setup classpath from current project

### ISSUE THREAD

**Issue by foal** [nileshkhaire/QuickShell#3]
Now JShell does not see the project dependencies and can't run any code from the project that depends on some third-party dependencies.

**Comment by nileshkhaire** [comment]
@foal Yes, you are right. This is not considered in this release. I am noting down this feature for future release.

**Comment by olfMombach** [comment]
I second this enhancement. Any plans on implementing?

**Comment by nileshkhaire** [comment]
I was thinking about this feature but what if there are multiple projector in workspace ? From which projects these dependencies should be loaded ?

So in short this feature will need more analysis.

Your suggestions are always welcome :)

**Comment by olfMombach** [comment]
Hi Nilesh, thanks for considering the suggestions!

For the class path loading, my idea would be that you could add a context menu entry to projects in the left explorer pane to "open this project configuration in jshell", that starts a jshell session using the project's run configuration's class path (I think that's possible with the --class-path flag for jshell). What do you think about the idea?
